src/seedsigner/models/seed.py

```diff
@@ -239,3 +239,61 @@ def seedqr_supported(self) -> bool:
     @property
     def bip85_supported(self) -> bool:
         return False
+
+
+
+class ShamirSeed(Seed):
+    def __init__(self,
+                 mnemonics: List[str] = None,
```

Comment:
The args are the same as the parent class except for `mnemonics` vs `mnemonic` so I'd opt for staying consistent, even if the idea of a mnemonic is a bit different for Shamir.

Reply:
Oh, wait, I see now. It's a list of space-separated mnemonic strings. 

```
['my first mnemonic phrase', 'my second mnemonic phrase']
```

Ug, I hate that format.